section3/exercises/ex39.rb

```diff
@@ -0,0 +1,66 @@
+# create a mapping of state to abbreviation
```

Comment:
Really nice job with these, good use of Ruby conventions - which makes your code very readable!